azure_li_services/defaults.py

```diff
@@ -30,6 +30,10 @@ class Defaults(object):
     def get_config_file_name(self):
         return '/etc/suse_firstboot_config.yaml'
 
+    @classmethod
+    def get_status_report_directory(self):
+        return '/etc/azure_li_services'
```

Comment:
I think this should be in /var/lib not /etc

Reply:
I was tempted to put these in `/var/log`, but they're not log files, they're YAML files.

Why are they YAML files? Who is expecting to read them? Is there a log that will give more details as well in case of problems? Say, for example, an invalid .RPM file exists, and can't be installed due to a conflict. How would we know that?

Reply:
>> Say, for example, an invalid .RPM file exists, and can't be installed due to a conflict.
>> How would we know that?
>
>systemctl status azure-li-install

But won't that be cleaned up by the cleanup service? It would seem that once the cleanup service completes, we won't have this sort of status anymore. So does that mean we're relegated to looking at YAML files to determine success/failure? Is there a "global" success/failure we can look at to avoid looking at each individual component?

What you're saying does make sense. I'd just like a way to:

1. Quickly/easily determine success/failure of overall YAML processing, and
1. Be able to see a detailed log of operations if something did go wrong. For example, if a specific RPM file wouldn't install, how would we know which one? (That's just an example, that's true for creation of accounts, mounting of disks, whatever.)

I'm signing off on this, as these are all details we can work out later, and shouldn't hold up an (otherwise) good commit.

Reply:
> But won't that be cleaned up by the cleanup service?

I don't think so. The cleanup service deletes the services so you can't recall them but the systemd daemon still has the cached information from the call which you still can retrieve until you reload the systemd daemon. I will give this a test to be 100% sure though